Autoloads/Globals.gd

```diff
@@ -4,15 +4,28 @@ extends Node
 @onready var MUSIC_BUS_ID = AudioServer.get_bus_index("Music")
 
 var user_prefs:UserPrefs
+var game_info: GameInfo
 var save:SaveData
 #var user_save:UserSave
 
 # temp - I'm not wild about preloads, but this menu is fairly light (to revise in a future version)
 var settings_menu_scene:PackedScene = preload("res://Menus/settings_menu.tscn")
 var settings_menu = null
  
+# available window resolutions
```

Comment:
What are your thoughts on moving the bulk of this window stuff into a window_info or similar sub-class? Obviously the initialization stuff in the _ready function would remain there in some form, as Globals is currently acting like the glue that packages up all these settings, but I'm not sure the resolutions array and other stuff belongs at the top level Globals doc? (figuring this out in real time, would love your thoughts). I try to keep code specific to features in specialized classes and then glue it together in a parent file/node, if that makes sense.

Reply:
moved to `Globals.display_manager.restore_window()`